L/LAST/build_tarballs.jl

```diff
@@ -0,0 +1,74 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message
+using BinaryBuilder, Pkg
+
+name = "LAST"
+version = v"1499"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource(
+        "https://gitlab.com/mcfrith/last.git",
+        "2cc68d3ba8ae5ca46ceeb69302aef18b9db04f46",
+    ),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/last
+install_license COPYING.txt
+export CXXFLAGS=" -O3 -Wall -g -std=c++11 -pthread"
+if [[ "${target}" != aarch64-* ]]; then
+    CXXFLAGS="${CXXFLAGS} -msse4"
+fi
+make CXXFLAGS="${CXXFLAGS}" -j${nproc}
+make bindir=${bindir} install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+# NOTE: Windows builds require Cygwin (sys/mman dependency)
+platforms = filter(
+    p -> !Sys.iswindows(p) && !any(arch(p) .== ["powerpc64le", "armv6l", "armv7l"]),
```

Comment:
Isn't
```suggestion
    p -> !Sys.iswindows(p) && !(arch(p) in ("powerpc64le", "armv6l", "armv7l")),
```
simpler (and non-allocating)? Or use `∉` if you fancy Unicode operators.